cider/exec/module/batch/CiderBatch.cpp

```diff
@@ -253,6 +253,51 @@ void CiderBatch::releaseArrowEntries() {
   }
 }
 
+// to be deprecated, just for test not nullable data.
+void CiderBatch::convertToArrowRepresentation() {
```

Comment:
This is converting to structbatch right? if column_num() is 1, should ciderbatch convert to strcutbatch or scalarbatch?

Reply:
yes. I think even a single column batch should convert to structBatch.

Reply:
Then what is the use of scalarbatch?

Reply:
I think codegen internal will never use scalarBatch. maybe it's just for test. Correct me if I am wrong. @bigPYJ1151